docs/DownloadBOLs.md

```diff
@@ -0,0 +1,133 @@
+---
+stoplight-id: jfj2qs9nubq5a
+---
+
+# Downloading Bills of Lading
+
+Get the full RunBuggy order details. Replace the text in curly brackets in the URL with your order ID.
```

Comment:
Please update to

```
1. Retrieve expanded order.

[Retrieve an Expanded Order](https://docs.runbuggy.com/docs/shipping-api/e37c8f68eee73-retrieve-an-expanded-order)
```

without the in-line example curl. It makes it easier for us to maintain docs by always pushing readers to dedicated pages where we can make sure the info is up-to-date.